site/src/lib/api/blocks/r2.ts

```diff
@@ -0,0 +1,97 @@
+import S3 from "aws-sdk/clients/s3";
+import fs from "fs-extra";
+import mime from "mime-types";
+import path from "node:path";
+
+import { isProduction } from "../../config";
+
+const cloudflareEndpoint =
+  "https://c6499786332a3d2fb35419a7803ab7aa.r2.cloudflarestorage.com";
```

Comment:
Do we need to hardcode this?

Reply:
nope - switched to env and documented new envs in https://github.com/blockprotocol/blockprotocol/pull/522/commits/db9a55ba883429bc77f70d061d2b78d51aba1e80